src/platform/forms-system/src/js/web-component-patterns/datePatterns.jsx

```diff
@@ -37,6 +41,17 @@ const currentOrPastDateUI = options => {
     'ui:options': {
       ...uiOptions,
     },
+    'ui:reviewField': ({ children }) => (
+      <Tag className="review-row">
+        <dt>{uiTitle}</dt>
+        <dd>
+          {new Date(`${children.props.formData}T00:00:00`).toLocaleDateString(
```

Comment:
do we want to manually specify the error here if formData doesn't exist? right now it seems to work - it will say "Invalid Date", but maybe we want to leave at blank instead?

Reply:
That's a super great point, I will default it to be blank